### PR TITLE
Homepage compare with MSAs

### PR DESCRIPTION
**Pull request by chasulin**
- [x] Fix sharing
- [x] Clean up

### CODE REVIEW

**Review by ghop02 — COMMENTED** [review]
Overall looking good! Added a few random suggestions

File: src/components/Compare/HomepageSlider.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  },
+  {
+    value: 99,
+    label: homepageLabelMap[HomepageLocationScope.STATES].plural,
+  },
+];
+
+const HomepageSlider: React.FC<{
+  onChange: any;
+  homepageScope: any;
```
[the remark this answers is not in the document]
Comment (chasulin) [reply src/components/Compare/HomepageSlider.tsx:39]: thats the plan

File: src/common/utils/compare.ts [hunk 2]
```diff
@@ -38,18 +38,18 @@ export const orderedMetrics = [
 ];
 
 function getLocationObj(region: Region): SummaryForCompare {
-  if (region.regionType === RegionType.COUNTY) {
+  if (region instanceof County || region instanceof MetroArea) {
```
[the remark this answers is not in the document]
Comment (chasulin) [reply src/common/utils/compare.ts:41]: oo yes i think so

File: src/common/utils/compare.ts [hunk 3]
```diff
@@ -198,12 +202,23 @@ export function getLocationPageViewMoreCopy(
   }
 }
 
+export enum HomepageLocationScope {
+  COUNTIES,
+  CITIES,
```
[the remark this answers is not in the document]
Comment (chasulin) [reply src/common/utils/compare.ts]: switched to `MSA`

File: src/common/utils/compare.ts [hunk 4]
```diff
@@ -198,12 +202,23 @@ export function getLocationPageViewMoreCopy(
   }
 }
 
+export enum HomepageLocationScope {
```
[the remark this answers is not in the document]
Comment (chasulin) [reply src/common/utils/compare.ts:196]: punting this to a larger restructure post-msas

**Review by pnavarrc — APPROVED** [review]
Looks good!

File: src/components/Compare/CompareMain.tsx [hunk 5]
```diff
@@ -99,20 +107,33 @@ const CompareMain = (props: {
   const [sortDescending, setSortDescending] = useState(true);
   const [sortByPopulation, setSortByPopulation] = useState(false);
   const [countyTypeToView, setCountyTypeToView] = useState(MetroFilter.ALL);
+
   // For homepage:
   const [viewAllCounties, setViewAllCounties] = useState(false);
-  // For location page:
-  const [geoScope, setGeoScope] = useState(GeoScopeFilter.STATE);
+  const [homepageScope, setHomepageScope] = useState(
```
[the remark this answers is not in the document]
Comment (chasulin) [reply src/components/Compare/CompareMain.tsx:113]: `viewAllCounties` is the old homepage prop that controlled states vs counties switch/view. it's passed down through a lot of components, and im putting off deleting all of them for a later PR

**Pull request merged**